apps/web-giddh/src/app/services/settings.integration.service.ts

```diff
@@ -625,4 +625,42 @@ export class SettingsIntegrationService {
             return data;
         }), catchError((e) => this.errorHandler.HandleCatch<any, any>(e)));
     }
+
+    /**
+     * Multi-purpose CRUD method for PayU integration.
+     * @param method HTTP method type ('GET' | 'POST' | 'PUT' | 'DELETE')
+     * @param payload Optional payload for POST/PUT/DELETE
+     * @returns Observable<BaseResponse<any, any>>
+     * @memberof SettingsIntegrationService
+     */
+    public payuCrudOperation(
+        method: 'GET' | 'POST' | 'PUT' | 'DELETE',
+        payload?: any
+    ): Observable<BaseResponse<any, any>> {
+        this.companyUniqueName = this.generalService.companyUniqueName;
+        const url = this.config.apiUrl + SETTINGS_INTEGRATION_API.PAYU_CRUD
+            ?.replace(':companyUniqueName', encodeURIComponent(this.companyUniqueName))
+
+        let httpCall: Observable<any>;
+
+        if (method === 'GET') {
```

Comment:
```suggestion
        this.http[method](url);
```

export enum ApiMethodEnum {
    GET = 'get',
    POST = 'post',
    PUT = 'put',
    PATCH = 'patch',
    DELETE = 'delete'
}